pilot/pkg/model/config.go

```diff
@@ -345,11 +344,12 @@ const (
 var (
 	// MockConfig is used purely for testing
 	MockConfig = ProtoSchema{
-		Type:        "mock-config",
-		Plural:      "mock-configs",
-		Group:       "test",
-		Version:     "v1",
-		MessageName: "test.MockConfig",
+		ClusterScoped: false,
```

Comment:
`ClusterScoped` defaults to false. We shouldn't need to set explicitly (see `Gogo` usage below).

Reply:
Done